### ISSUE TITLE
Access "Total play time" from MTP

### ISSUE THREAD

**Issue by profucius** [rashevskyv/dbi#88]
Hi there, great tool. I currently use MTP to make "installed games" available to my windows computer, which is running a game management software called Playnite to keep track of all my installed games on the Switch, as well as other consoles, etc.

I would like to be able to extract the "Total play time" for games on the Switch and export them to the Playnite software. I don't mind handling the automation/scripting of processing the data. I wonder if you could create a feature within DBI that makes the "Total play time" for each game available within the MTP.

One way I have thought of this would be within the "Installed games" drive within the MTP on a windows computer. Currently, when you navigate into "Installed games" it shows a list of all the games, and when you navigate into a game folder, currently you see the NSP for the game as well as Mods & Cheats. I envision another (automatically generated) file within the game folder that is a TXT or CSV or JSON file that contains some specific details for the game that you can see within DBI when looking at the games details, which includes "Total play time".

An alternative method to this, or in addition to this, at the root of the MTP available on a computer (beside of "External SD Card", "Installed games", "NAND Install", etc. There could be a new entry here called "Game Details" or something like that. Within it would be separate files specifically for details. For instance, a file named "Total play time" that is an auto-generated CSV or JSON file with only the game name and the total play time as separate columns.

These methods would give me the data necessary to create an automated method to export data like "Total play time" and import it with minimal manual labor into third-party game play time tracking software such as Playnite.

Let me know if you have any questions or wish for me to clarify anything. Thanks!

**Comment by profucius** [comment]
TL;DR

I'd like a feature in DBI that is a menu entry, which generates a text (or excel?) file of the list of all games, sorted by played recently, including the amount of (unix) total gametime they were played for.

---

Another app already tried to do this (NX Activity Log) however that dev abandoned the app, and it continues to suffer incompatibility with newer firmwares. Plus, I would rather have access to this information directly from DBI when I connect to my PC.

I use this gametime information for syncing to my Playnite app on PC, so I can keep track of all my videogames playtime, this would enable tracking my switch games in the same app.
